src/pages/experimental/color/index.page.tsx

```diff
@@ -0,0 +1,66 @@
+import { ThemeConfig } from 'tailwindcss/types/config.js'
+import * as a from '../../../../tailwind.config.js'
+
+export default function Colors() {
+  if (!a.theme) return null
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const swathe = themeToSwathe(a.theme as any)
```

Comment:
**suggestion** `swatch` perhaps? :trollface:

Reply:
btw this is so sick

Reply:
Oh lol, I didn't mean to put this in the code base, but fuckit. It's already here lol